app/src/main/java/com/jnj/vaccinetracker/participantflow/screens/ParticipantFlowMatchingFragment.kt

```diff
@@ -32,7 +33,7 @@ import kotlinx.coroutines.flow.onEach
 class ParticipantFlowMatchingFragment : BaseFragment() {
 
     private companion object {
-        private const val REQ_REGISTER_PARTICIPANT = 453
+        private const val REQ_REGISTER_PARTICIPANT = Constants.REQ_REGISTER_PARTICIPANT
```

Comment:
the same as above, we don't need this assigning, we can just use the Constants.REQ_REGISTER_PARTICIPANT without REQ_REGISTER_PARTICIPANT const